books/tests/test_admin.py

```diff
@@ -0,0 +1,17 @@
+import pytest 
+from django.contrib.admin.sites import AdminSite
+from mixer.backend.django import mixer
+from django.db import models
+
+pytestmark = pytest.mark.django_db
+
+from .. import admin
+from .. import models
+
+class TestBookAdmin:
+    def test_display_author(self):
+        site = AdminSite()
+        book_admin = admin.BookAdmin(models.Book, site) 
+        obj = mixer.blend('books.Book', author="Kate Raworth")
+        display_author = book_admin.display_author(obj)
+        assert display_author == "Raworth, Kate", "Should return author with lastname first"
```

Comment:
Things to have in mind as tests start building up:

Consider putting the "message" portion in the title of the tests. Ex. this test could have been named `test_display_author__returns_last_name_first`.

Then you can have more tests like `test_display_author__moves_only_last_name_component` for example if you wanted to test that a multi-component name like "Carla Rodriguez Calderón" becomes "Calderón, Carla Rodriguez".

Generally you would have a separate test for each scenario, so it's useful to describe the test in its definition.

Reply:
okay. I put the message there, because in a video somebody said they always used them :), but of course if test name describes purpose, you don't need a message